### ISSUE TITLE
Generate code for custom-baseurl.json

### ISSUE THREAD

**Issue by jianghaolu** [Azure/autorest.java#421]

**Comment by weidongxu-microsoft** [comment]
Need upgrade modelerfour.

4,0.52 incorrectly have `in: path`

4.1.60 would have `in: url`
```
globalParameters:
- &ref_3
  schema: *ref_2
  implementation: Client
  required: true
  language:
    default:
      name: accountName
      description: Account Name
  protocol:
    http:
      in: uri
- &ref_4
  schema: *ref_2
  clientDefaultValue: host
  implementation: Client
  required: true
  language:
    default:
      name: host
      description: A string value that is used as a global part of the parameterized host
  protocol:
    http:
      in: uri
```

**Comment by jianghaolu** [comment]
Closed via #438.
